stacks/spica/packages/material/save/save.directive.ts

```diff
@@ -0,0 +1,47 @@
+import {Directive, Input, TemplateRef, ViewContainerRef} from "@angular/core";
+
+export enum SavingState {
+  Pristine = "",
+  Saving = "Saving",
+  Saved = "Saved",
+  Failed = "Failed"
```

Comment:
```suggestion
  Saving = "saving",
  Saved = "saved",
  Failed = "failed"
```
Let's replace the enum values with lowercase ones.